test/Makefile.am

```diff
@@ -0,0 +1,3 @@
+test:
+	$(MAKE) -C ../src/retrace config.py
+	PATH=${abs_top_srcdir}/src:$(PATH) RETRACE_SERVER_PLUGIN_DIR=${abs_top_srcdir}/src/plugins RETRACE_SERVER_CONFIG_PATH=${abs_top_srcdir}/src/config/retrace-server.conf python run_test.py --coredump=$(coredump) --executable=$(executable) --package=$(package) --release=$(release) $(ARGS)
```

Comment:
If you just want to run the test, the test should set up the environment itself. It should not be needed to pass those arguments on command line.